Train.py

```diff
@@ -22,12 +22,29 @@
 criterion = torch.nn.MSELoss().cuda()
 optimizer = torch.optim.Adadelta(net.parameters())
 
+"""
```

Comment:
We are currently making the transition to saving and resuming the train loss saves using the standard csv format. I already opened a pull request with this but @tpankaj is making some changes before it will be merged in. For now, I think the current untested resuming method without saving implemented should be removed from the codebase.